README-Python-Dev.md

```diff
@@ -0,0 +1,76 @@
+# Developing a Python dependency of this project
+
+This document contains a suggested approach on how to develop or modify a Python library that uses the current project Docker as a test or develop environment.
+For example, those instructions can be used to create a developer environment for further development of [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python).
+The instructions below are intended to be a suggestion or a guideline rather than a standard.
+
+## Step 0: Identify which Python dependency requires development
+
+We assume that the developer needs to develop a modified version of the library [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python) which is a dependency of the container `satosa-saml2spid`.
+A local copy of the library is required.
+We assume that the project eudi-wallet-it-python has been cloned in the folder `/home/username/my/development/folder/eudi-wallet-it-python/pyeudiw`. The path prefix `/home/username/my/development/folder/` is an example and should be replaced here with the location of your own development package.
+
+## Step 1: Set environment variable
+
+Set the environment variable `SATOSA_DEBUG=true`. This can be done either in the terminal with the command `export SATOSA_DEBUG=true`, or by updating the file [.env](Docker-compose/.env) by appending the entry `SATOSA_DEBUG=true`.
+
+## Step 2: Update the docker volume by binding the local development directory
+
+In the file [docker-compose.yml](Docker-example/docker-compose.yml), among the volumes of the container `satosa-saml2spid`, add the entry
+    
+        volumes:
+            - /home/username/my/development/folder/eudi-wallet-it-python/pyeudiw:/.venv/lib/python3.12/site-packages/pyeudiw:rw
+
+This will replace the installed dependency package with your own local code.
+
+**NOTE:** at the time of writing, container volume is binded to the location `/.venv/lib/python3.12/site-packages`, but your location might be different as it always reference the Python version that is awailable in the container, which in this case is `Python3.12`. Check che actual python version of your container before completing with this step.
+
+## Step 3: Run the container
+
+Launch the script [run-docker-compose.sh](Docker-compose/run-docker-compose.sh). This will launch the docker composition that includes the container `satosa-saml2spid`.
+
+## Step 4 (Optional): Install further dependencies in the container
+
+If your version of the library containes further dependency, or if you want to install development only dependency such as, say [pdbpp](https://github.com/pdbpp/pdbpp), you can create a new image that contains the required dependency.
```

Comment:
Note that `docker container commit` command actually creates a new image (see the [docs](https://docs.docker.com/reference/cli/docker/container/commit/#:~:text=Description,a%20container%27s%20changes)) hence the proposed addition is technically redundant.

Since this might not be a well known detail of the command, I am happy to include the proposed addition.